src/components/notification/process-server-status.ts

```diff
@@ -156,9 +158,12 @@ export const processThresholds = ({
       alertQuery: alert.query,
       state: state.value as 'UP' | 'DOWN',
       shouldSendNotification:
+        isFirstTimeSendEvent ||
```

Comment:
I think it will send a recovery notification if the first event is UP, could you check it again?

Reply:
Nope, it will send even the first event DOWN

<img width="420" alt="Screen Shot 2022-02-25 at 15 42 54" src="https://user-images.githubusercontent.com/20070808/155683818-b3171f44-72d2-42eb-b783-7135d6ac4fe8.png">

Reply:
https://user-images.githubusercontent.com/15191978/155685964-6610f540-fdb1-442f-b8a1-8a7a1e84baf4.mp4